### ISSUE TITLE
ptipython doesn't work

### ISSUE THREAD

**Issue by lusu2004** [prompt-toolkit/ptpython#319]
1. anaconda for python 3.7
2. pip install ptpython
3. ptipython
4. input below command
   %pylab   #it will use matplotlib backend: Qt5Agg
   x = np.linspace(0,6,1000)
   y= np.sin(x)
   plt.plot(x,y)

the Figure1 window rised, but show nothing.

Thanks

**Comment by yamsu** [comment]
Yes I can confirm this. However, the same lines of code are working fine in vanilla ipython

**Comment by yamsu** [comment]
This seems to be an old issue.
Please refer to issues #20 #221 

From a cursory reading its looks like the issue is with the event loops for ptipthon and ipython.
